far/filelist.cpp

```diff
@@ -982,7 +982,7 @@ long long FileList::VMProcess(int OpCode,void *vParam,long long iParam)
 					if (i.empty())
 						continue;
 
-					const auto Pos = FindFile(PointToName(i), true);
+					const auto Pos = FindFile(i, !wcschr(namePtr,LGOOD_SLASH));
```

Comment:
It won't compile.
`!wcschr(namePtr,LGOOD_SLASH)` -> `!i.contains(path::separator)`.


This is still a bit incompatible: previously it was possible to select names like `"q:\non_existing_or_completely_unrelated_path\file.ext"`, as long as there is `file.ext` on the panel, but we can probably live with that.

Reply:
> but we can probably live with that.

Hmm.. I rather see that as breaking change